db/crud.py

```diff
@@ -0,0 +1,54 @@
+from sqlalchemy.orm import Session
+
+import models
+import schemas
+
+
+def get_all_books(db: Session):
+    return db.query(models.DBBook).all()
+
+
+def get_book_by_title(db: Session, title: str):
+    return (
+        db.query(models.DBBook).filter(models.DBBook.title == title).first()
+    )
+
+
+def create_book(db: Session, book: schemas.BookCreate):
+    db_book = models.DBBook(
+        title=book.title,
+        summary=book.summary,
+    )
+    db.add(db_book)
+    db.commit()
+    db.refresh(db_book)
+
+    return db_book
+
+
+def get_author_list(
+    db: Session,
+    books: str | None = None,
+):
+    queryset = db.query(models.DBAuthor)
+
+    if books is not None:
+        queryset = queryset.filter(models.DBAuthor.book.has(title=books))
```

Comment:
The `filter` method uses `models.DBAuthor.book.has(title=books)`, but the `DBAuthor` model has a relationship named `book`, not `books`. Ensure that the field name matches the relationship defined in the `DBAuthor` model.